requirements-github.txt

```diff
@@ -1,14 +1,14 @@
 pyyaml>=6.0
-pycodestyle==2.9.1
-netCDF4==1.6.1
+pycodestyle>=2.9.1
```

Comment:
what are the exact versions that get downloaded now? Can we use those instead?

Reply:
I was following the changes that were made to EVA while I was debugging. We can but I am not sure what the exact versions are. Where can I find that information?

Reply:
meh, if that's what Akira/Dan did then this is fine for now.